profile-dotemacs.el

```diff
@@ -72,13 +72,16 @@
 
 ;;; Code:
 
+(require 'cl)
 (require 'thingatpt)
 (require 'benchmark)
 
 ;; User variables
 
-(defvar profile-dotemacs-file "~/.emacs.d/init.el"
-  "File to be profiled.")
+(defconst profile-dotemacs-user-init-files '("~/.emacs"
```

Comment:
I prefer to use `defvar` rather than `defconst` since it increases customizability.

Reply:
Sure. I wasn't sure why the other repository chose `defconst`. Is the use of `defvar` best practice or are there cases one would chose `defconst`?

Reply:
I think `defvar` should be preferred. Using `defconst` has some documentary value ("you shouldn't change this variable"), and also allows the byte-compiler to inline its uses. This improves efficiency slightly at the cost of no longer allowing the variable to be changed or dynamically bound.